app/services/api/trainees/award_recommendation_service.rb

```diff
@@ -21,6 +21,8 @@ class AwardRecommendationService
 
       validates_with AwardRecommendationValidator
 
+      validate :trainee_has_degrees?
```

Comment:
I think we need to check the business logic that all trainees will require degrees, even those on UG routes.

Reply:
The policy steer on this is that we only need to require degrees for non UG routes. This is the [current logic](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/lib/training_route_manager.rb#L10) in the `TrainingRouteManager` so we should match this for this API endpoint.